docs/source/guides/configuration.rst

```diff
@@ -27,6 +27,7 @@ at ``example/config.sample``
     "vote_delay": 5000,
     "mst_enable" : false,
     "mst_expiration_time" : 1440,
+    "mst_stalled_batch_threshold" : 1800000,
```

Comment:
```suggestion
    "mst_stale_threshold_milliseconds" : 1800000,
```
the related file is https://github.com/hyperledger/iroha/blob/ee6b906daade545019e331f00cc7b68a81507d98/irohad/main/iroha_conf_literals.cpp#L26

please consider other places too

Reply:
Thanks! please check if everything is ok now!